recipes/muparser/build.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+sh ./configure --prefix=$PREFIX --disable-samples
+make -j2
```

Comment:
@183amir, why would it be buggy ? it's Makefile has no problem with parallel builds. Why 2 cores ? travis and circleci give us 2 cores, I just use them.